index.js

```diff
@@ -290,7 +332,7 @@ var processEvent = function(event, context) {
     slackMessage = handleAutoScaling(event, context);
   }
   else{
-    context.fail("No matching processor for event. [EventSubscriptionArn: " + eventSubscriptionArn + "]");
+    slackMessage = handleCatchAll(event, context);
```

Comment:
This should work, but if we ever have any issues we can wrap it in a `try/catch` to call `context.fail` about the failure. Nothing to do rn, just a note.